webots_ros2_core/webots_ros2_core/devices/laser_device.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 1996-2020 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Laser device."""
+
+from sensor_msgs.msg import LaserScan
+from rclpy.time import Time
+from geometry_msgs.msg import TransformStamped
+from tf2_ros import StaticTransformBroadcaster
```

Comment:
Unfortunately, this is not available on ROS2 Dashing and before (https://answers.ros.org/question/309889/ros2-modulenotfounderror-no-module-named-tf2_ros/), but we can maybe decide to drop Dashing as supporting the two last version of ROS2 is already good IMO.

Reply:
`StaticTransformBroadcaster` only defines QoS suitable for static transforms, not hard to avoid using it.

Reply:
If it is easy ok, otherwise (just discussed with @omichel), dropping support for Dashing, is fine (95% of the packages have moved to Eloquent).